frontend/device/kobo/device.lua

```diff
@@ -97,6 +106,10 @@ function Kobo:init()
         )
     end
 
+    if self.touch_alyssum_protocol then
```

Comment:
Ah, there really has been a lot of changes in the code. This used to be in input:init. This refactored code is much easier to read through, though!